fineract-provider/src/integrationTest/java/org/apache/fineract/integrationtests/common/HolidayHelper.java

```diff
@@ -77,15 +77,15 @@ public static Integer createHolidays(final RequestSpecification requestSpec, fin
 
     public static Integer activateHolidays(final RequestSpecification requestSpec, final ResponseSpecification responseSpec,
             final String holidayID) {
-        final String ACTIVATE_HOLIDAY_URL = HOLIDAYS_URL + "/" + holidayID + "?command=activate&" + Utils.TENANT_IDENTIFIER;
-        return Utils.performServerPost(requestSpec, responseSpec, ACTIVATE_HOLIDAY_URL, getActivateHolidayDataAsJSON(), "resourceId");
+        final String activateHolidayURL = HOLIDAYS_URL + "/" + holidayID + "?command=activate&" + Utils.TENANT_IDENTIFIER;
+        return Utils.performServerPost(requestSpec, responseSpec, activateHolidayURL, getActivateHolidayDataAsJSON(), "resourceId");
     }
 
     public static HashMap getHolidayById(final RequestSpecification requestSpec, final ResponseSpecification responseSpec,
             final String holidayID) {
-        final String GET_HOLIDAY_BY_ID_URL = HOLIDAYS_URL + "/" + holidayID + "?" + Utils.TENANT_IDENTIFIER;
+        final String getHolidayByIDurl = HOLIDAYS_URL + "/" + holidayID + "?" + Utils.TENANT_IDENTIFIER;
         LOG.info("------------------------ RETRIEVING HOLIDAY BY ID -------------------------");
```

Comment:
Would be good to be consistent with capitalising URL - so this would be getHolidayByIdURL

Reply:
upated